backend/readers_backend/readers_backend/settings.py

```diff
@@ -173,11 +173,13 @@ def get_required_env_var(key):
 
 # Email Configuration for production
 EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
-EMAIL_HOST = os.environ.get("EMAIL_HOST", "smtp.gmail.com") # Gmail's SMTP server
-EMAIL_PORT = int(os.environ.get("EMAIL_PORT", 587)) # Gmail's SMTP port
-EMAIL_USE_TLS = os.environ.get("EMAIL_USE_TLS", "True") == "True" # TLS is required for Gmail 
+EMAIL_HOST = os.environ.get("EMAIL_HOST", "smtp.gmail.com")  # Gmail's SMTP server
```

Comment:
Next time, stage changes by file (or even patch hunk), and omit files your formatter messed with, but you didn’t. It just clutters the commit.